### ISSUE TITLE
Not existed data after inserting them in a previously deleted metric on VM

### ISSUE THREAD

**Issue by mr-yaky** [VictoriaMetrics/VictoriaMetrics#1438]
We started to receive very strange issue with VictoriaMetrics even with different versions for all our environments. We get that problem when we are inserting data in a previously deleted metric on VM.
You can find all details bellow.

**Summary**:
On VictoriaMetrics sometimes when we delete some metrics and then we insert the data again with same metric name, it doesn't work for some random tags.

**Steps to reproduce:**
Delete a list of metrics (around 50) with a list of tags (43 tags), like this:
```
axios.post(VictoriaConfig.deleteURL(), null, {
    params: {
                'match[]': `${signal}{_id_turbine=~"${turbines}"}`
            }
    })
```
After waiting for half an hour, we start inserting data again for all these metrics and tags (from 1st of january until now) in insertion bulks. Everything looks ok at this point, we can see the data there with no gaps.
Then from this point on, we insert one datapoint per metric/tag every ten minutes aprox.

**What is the current bug behavior?**
After one day working as expected, we can see for some metrics with specific tags, the data is not inserted anymore, but we get no error from Victoria, just 204 code as usual.

![image (5)](https://user-images.githubusercontent.com/18198525/124951958-20d91f00-e014-11eb-8d09-d20c4331f250.png)

in this example we represent 2 metrics per graph and each graph belongs to one _id_turbine tag:

case 1: both metrics are wrong for this tag.
case 2: both metrics are ok for this tag.
case 3: only one metric is wrong for this tag.

**What is the expected correct behavior?**
When we delete one metric we should be able to still inserting datapoints with same metric name, it should not discard some of them them randomly. As Victoria doesn't allow data updates, we should have a safe way to delete metrics without affecting incoming inserts.

**Version**
We are using both version on different environments:

v1.61.1-cluster
v1.54.1-cluster

**Comment by f41gh7** [comment]
Thanks for detailed issue description. Need to investigate it.

**Comment by valyala** [comment]
This issue can be related to https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1347 . This bug should be fixed in [v1.62.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). @mr-yaky , could you upgrade all the VictoriaMetrics cluster components to `v1.62.0` and verify whether this fixes the issue?

**Comment by mr-yaky** [comment]
Yes, this was exactly that issue and we fixed it with manual removing cache on v1.61.1. We will check later with v1.62.0 and I'll write you the feedback.

**Comment by denisgolius** [comment]
> Yes, this was exactly that issue and we fixed it with manual removing cache on v1.61.1. We will check later with v1.62.0 and I'll write you the feedback.
@mr-yaky note that there is a new version of VictoriaMetrics - https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.63.0

**Comment by mr-yaky** [comment]
Thanks @denisgolius , I meant about v1.62.0 as the version which contains the fix for auto-purging cache for this issue after the starting of VM process.
